class7-week3/actions/clear.js

```diff
@@ -0,0 +1,16 @@
+const Todo = require('../models/todo')
+
+module.exports = function create(request, response) {
```

Comment:
Good thinking to copy `create` - but you should rename it to `clear`. Also would have been easier to copy `remove`.